LICENSE

```diff
@@ -0,0 +1,339 @@
+                    GNU GENERAL PUBLIC LICENSE
+                       Version 2, June 1991
```

Comment:
I would use version 3.

You choosing the modern v3 is compatible with MDA's old v2 because we allow "or any later version".